### PR TITLE
bug fix on wsdl2jolie

### PR DESCRIPTION
**Pull request by klag**
could we check the status of this pull request?

### CODE REVIEW

**Comment by fmontesi** [comment]
What does this fix? There are also changed in jolie2dummy.

**Comment by klag** [comment]
it fixes a bug in wsdl2jolie conversion.

Simple types inside subelements of a complex type they were not transformed.

jolie2dummy has been improved. I did it separately, but the pull request
aggregated them.



2016-12-16 10:58 GMT+01:00 Fabrizio Montesi <notifications@github.com>:

File: extensions/auto/src/jolie/net/auto/AutoChannelFactory.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import jolie.runtime.AndJarDeps;
+
+/**
+ *
+ * @author Claudio Guidi
+ */
+@AndJarDeps({"ini4j.jar"})
+public class AutoChannelFactory extends CommChannelFactory {
+    
+    CommCore commCore;
```
Comment (fmontesi) [inline extensions/auto/src/jolie/net/auto/AutoChannelFactory.java]: private final CommCore

File: extensions/auto/src/jolie/net/auto/AutoChannelFactory.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import jolie.net.ext.CommChannelFactory;
+import jolie.net.ports.OutputPort;
+import jolie.runtime.AndJarDeps;
+
+/**
+ *
+ * @author Claudio Guidi
+ */
+@AndJarDeps({"ini4j.jar"})
+public class AutoChannelFactory extends CommChannelFactory {
```
Comment (fmontesi) [inline extensions/auto/src/jolie/net/auto/AutoChannelFactory.java]: I still see a different indentation style, the { should be in the next line.

File: extensions/auto/src/jolie/net/auto/AutoChannelFactory.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    
+    CommCore commCore;
+    HashMap<URI,String> locationMap = new HashMap<URI,String>();
```
Comment (fmontesi) [inline extensions/auto/src/jolie/net/auto/AutoChannelFactory.java]: private final Map< URI, String > locationMap

File: extensions/auto/src/jolie/net/auto/AutoHelper.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+import jolie.util.Helpers;
+import org.ini4j.Ini;
+
+/**
+ *
+ * @author claudio
+ */
+public class AutoHelper {
+    
+        public static IOException buildIOException( String message )
```
Comment (fmontesi) [inline extensions/auto/src/jolie/net/auto/AutoHelper.java]: Indentation (maybe your netbeans is not applying anymore the conf file I passed once for automatic source formatting?).

**Pull request merged**